resources/build.sh

```
@@ -11,20 +11,21 @@
-  bash) source "$CONDA_PREFIX/share/harpy/complete.bash" ;;
-  zsh)  source "$CONDA_PREFIX/share/harpy/complete.zsh"  ;;
-  fish) source "$CONDA_PREFIX/share/harpy/complete.fish" ;;
-esac
-EOF
\ No newline at end of file
+#echo "export JUPYTER_NOTARY_DB=':memory:'" > ${PREFIX}/etc/conda/activate.d/my-package-activate.sh
+#echo "unset JUPYTER_NOTARY_DB" > ${PREFIX}/etc/conda/deactivate.d/my-package-deactivate.sh
+cat > ${PREFIX}/etc/conda/activate.d/harpy-activate.sh <<'EOF'  
+export _HARPY_OLD_JUPYTER_NOTARY_DB="${JUPYTER_NOTARY_DB-__UNSET__}"  
+export JUPYTER_NOTARY_DB=':memory:'  
+EOF  
```

Shellcheck (0.11.0)

[error] Delete whitespace after the here-doc end token.

(SC1118)